### DISCUSSION TITLE
Interpretation of  TagBamWithReadSequenceExtended summary file

### DISCUSSION THREAD

**Discussion by mytlsdnkr** [broadinstitute/Drop-seq#303] [Q&A] [answered]
Hello.

I'm user of drop-seq tools, thanks for your sharing about this good software.

I have a some question of interpretation of summary file in TagBamWithReadSequenceExtended result.

I used option that ranged base from 1-12 bp in my single cell data.

But, in this summary file range of bp was 0-12, so, what is meaning of 0? and what is the meaning of each column?

Thanks for your answer!

Regards!

![image](https://user-images.githubusercontent.com/20651997/166092060-8d670e57-81c7-4fd5-8f53-549c100c0d9e.png)

**Comment by jamesnemesh** [comment, marked as answer]
Hi,

The histogram tells you how many bases had a quality score lower than your threshold. It’s possible every base would have a quality score lower than your threshold, so the maximum value is the number of bases in your barcode. However, it’s possible that no bases in your barcode were below the quality score threshold, so in that case the number of bases is 0.

This output does not refer to the position at which a low quality base was detected.

-Jim
